### PR TITLE
Update manifests and templates to 2.2.2

### PR DESCRIPTION
**Pull request by jewertow**
Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

### CODE REVIEW

**Review by luksa — APPROVED** [review]

File: manifests-maistra/2.2.2/maistraoperator.v2.2.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,777 @@
...
+      [
+        {
+          "apiVersion": "maistra.io/v2",
+          "kind": "ServiceMeshControlPlane",
+          "metadata": {
+            "name": "basic",
+            "namespace": "control-plane-namespace"
+          },
+          "spec": {
+            "version": "v2.2",
```
Comment (jewertow) [inline manifests-maistra/2.2.2/maistraoperator.v2.2.2.clusterserviceversion.yaml:33]: @luksa  I updated SMCP versions here and in 2.2.0 and 2.2.1 as well, but gen-check fails only for 2.2.2. Do you know why?
Btw. should it be updated manually or is it somehow automatically?

Comment (luksa) [reply manifests-maistra/2.2.2/maistraoperator.v2.2.2.clusterserviceversion.yaml:33]: You need to fix this in build/manifest-templates/clusterserviceversion.yaml

Comment (luksa) [reply manifests-maistra/2.2.2/maistraoperator.v2.2.2.clusterserviceversion.yaml:33]: Oops, sorry.. this is the 2.3 branch, not 2.2. So, you need to fix this in the maistra-2.2 branch and then an automatic PR should pull in the change.

Comment (jewertow) [reply manifests-maistra/2.2.2/maistraoperator.v2.2.2.clusterserviceversion.yaml:33]: Thanks!

**Pull request merged**